sr_robot_launch/config/left_bimanual_ur10e_arm_robot_hw.yaml

```diff
@@ -51,7 +51,7 @@ la_sr_ur_robot_hw:
   
   output_recipe_file: /home/user/projects/shadow_robot/base/src/Universal_Robots_ROS_Driver/ur_robot_driver/resources/rtde_output_recipe.txt
   
-  script_file: /home/user/projects/shadow_robot/base/src/Universal_Robots_ROS_Driver/ur_robot_driver/resources/ros_control.urscript
+  script_file: /opt/ros/noetic/share/ur_client_library/resources/external_control.urscript
```

Comment:
Seeming as we are already running off a fork of `Universal_Robots_ROS_Driver`, I wonder if it would be better to set the paths in these config files to `/home/user/projects/shadow_robot/base/src/Universal_Robots_ROS_Driver/ur_robot_driver/resources/external_control.urscript` and copy the current `external_control.urscript` file from [here](https://github.com/UniversalRobots/Universal_Robots_Client_Library/blob/2e69bc165da0ad22fe9a1a3417e7eaa4b082ebac/resources/external_control.urscript) to [this folder](https://github.com/shadow-robot/Universal_Robots_ROS_Driver/tree/master/ur_robot_driver/resources)

Reply:
Decision made to stick with the `/opt/ros/noetic/...` path in this PR